static/src/javascripts/projects/commercial/modules/header-bidding/prebid/permutive-test.ts

```diff
@@ -0,0 +1,19 @@
+import config_ from '../../../../../lib/config';
+
+// This is really a hacky workaround ⚠️
+const config = config_ as {
+	get: (s: string, d?: string) => string;
+};
+
+let isInTest: boolean | undefined;
+
+const isInServerSideTest = (): boolean =>
+	config.get('tests.prebidWithPermutiveVariant') === 'variant';
+
+export const isInPrebidPermutiveTest = (): boolean => {
+	if (isInTest === undefined) {
```

Comment:
Any reason for the memoization of the config value?
Is there a performance cost on calling config.get?
If not then we could return the result of `config.get` from `isInPrebidPermutiveTest`

Reply:
I think it might be premature optimisation.